C7/MainMenu.tscn

```diff
@@ -89,8 +89,8 @@ window_title = "Open a File or Directory"
 resizable = true
 mode = 3
 access = 2
-current_dir = "/Users/jim/code/src/godot/Prototype/C7"
-current_path = "/Users/jim/code/src/godot/Prototype/C7/"
+current_dir = "/Development/Civ Related Projects/Prototype/C7"
+current_path = "/Development/Civ Related Projects/Prototype/C7/"
```

Comment:
These variables shouldn't be modified as part of this PR, but really they shouldn't have been included in MainMenu.tscn in the first place. I tried it myself and found that they will get automatically reset to match the current system each time the game is launched. I think this started with the "Static variable in Util" PR that was merged recently. Could we avoid storing these, or at least move them to C7.ini?

Reply:
You're right, they shouldn't be there.  I have removed them in my new commit.  But they come back every time I start the program... they seem to be the path where the project lives (mine is what it states, except with C: in front).

What I'm not sure of is where they're coming from.  I don't see any code references that would generate them, nor do I see any changes in https://github.com/C7-Game/Prototype/pull/78/files that seem to explain it.

Reply:
I did some investigation and figured out where those variables are coming from. They belong to the Godot FileDialog node named "SetCiv3HomeDialog" that was added by the "Static variable in Util" PR. Have a look at the blame for MainMenu.tscn: https://github.com/C7-Game/Prototype/blame/Development/C7/MainMenu.tscn. You can find those variables in the editor if you look under the "Inspector" tab for that node. There's a little arrow next to each one that you can click to reset them to the current C7 project directory, and what must be happening is that Godot is resetting them each time the project is built.

I don't know what's the best way to fix this. We could create the file dialog at run time in C# code, that would work but it's not very elegant. Maybe there's a switch somewhere we could flip to stop Godot from resetting the vars. I search online briefly and didn't find anything.

Reply:
Anyway this isn't really related to the PR. I'll open an issue regarding these variables. The PR itself looks good.

Reply:
Interesting.  I see what you mean now that I pull it up in Godot's editor.  I checked to see why the BIQ/SAV dialog doesn't have the same issue, and it's instantiated in code.  I agree with putting it in a separate issue, I'll add it to Carthage.